src/library-authoring/component-picker/SelectLibrary.tsx

```diff
@@ -75,7 +77,7 @@ const SelectLibrary = ({ selectedLibrary, setSelectedLibrary }: SelectLibraryPro
   return (
     <Stack gap={2} className="p-5">
       <small className="text-primary-700">
-        {intl.formatMessage(messages.selectLibraryInfo)}
+        {intl.formatMessage(messages.selectLibraryInfo, { itemType })}
```

Comment:
(optional) (nit/minor) I know this was already like this, but we generally prefer to use the declarative `<FormattedMessage />` component over the imperative `intl.formatMessage()` API when there's a choice.